### ISSUE TITLE
support geomorphic boards with rotated unit boards

### ISSUE THREAD

**Issue by wsu-cb** [CyberBoardPBEM/cbwindows#50]
For hex unit boards, rotating 180 degrees should be possible.

For (future) square unit boards, rotating 90, 180, and 270 degrees should be possible, assuming the unit board is square, 180 degrees if the unit board is rectangular..

**Comment by DLLarson** [comment]
Makes sense. My original use case for creating the geomorphic feature was Avalon Hill's Advanced Squad Leader. https://en.wikipedia.org/wiki/Geomorphic_mapboard

-Dale
